### DISCUSSION TITLE
Is it really not possible to software raid1 with the SO 2.3.10 CentOS ISO?

### DISCUSSION THREAD

**Discussion by w4rc0n** [Security-Onion-Solutions/securityonion#2283] [General] [answered]
I could not find a way to do this in the installer. Even booting into single user mode just brought me into the Security Onion installer. 

Seems like kind of a huge oversight to not allow for software raid on mission critical systems. 

Any advice on how this could be accomplished?

**Comment by trb143** [comment]
Start the installer and then cancel out.  This returns you to the command prompt.  I have usually done this when I have entered the wrong option and wish to restart the installer.  
You will need to log out and in to restart the installer but you can make your changes first.

**Reply by w4rc0n** [reply]
Would you do that in the graphical mode, or the simple text mode? In both of them, I get to the screen where it asks what disk to install to and it wont accept any other input than what it expects aka disk name. If I get to the screen where it asks if I want to proceed, the only input I've found that it accepts is 'yes'.

**Reply by trb143** [reply]
You need to do the iso install to Centos, then reboot before you cancel out.

**Reply by w4rc0n** [reply]
Are you talking about the installer that is the so-setup-network script? Or the actual installer via the ISO? If I install to CentOS, then I would have to created a raid array after I do the CentOS install, which is not desirable. Much easier to just set up the raid array in the installer, then install onto that array. Like pretty much every distro except this one allows for.

**Comment by dougburks** [comment, marked as answer]
If you're wanting to use the standard CentOS installer, then you can always install standard CentOS 7 and then add our components on top of that as described here:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

**Reply by w4rc0n** [reply]
The entire reason I'm using your pre baked ISO in the first place is because installing the components on top of Ubuntu was a nightmare of random installation errors. I was able to work through many of them after many hours. But reached a salt error i couldn't get past that I posted about here and got no responses.
